plugins/inputs/opcua/opcua_client_test.go

```diff
@@ -75,6 +78,10 @@ func MapOPCTag(tags OPCTags) (out NodeSettings) {
 }
 
 func TestConfig(t *testing.T) {
+	if testing.Short() {
+		t.Skip("Skipping integration test in short mode. You can remove this skip after Init() doesn't try to connect")
+	}
+
```

Comment:
It seems to be failing because it's getting a `connection refused` error from GetEndpoints which is called by opcua under the hood of setupOptions in the Init.